blocks/library/freeform/style.scss

```diff
@@ -0,0 +1,12 @@
+.blocks-freeform .blocks-editable__tinymce {
+  ul, ol {
```

Comment:
Good catch. I have my editor in spaces mode by default - I've changed the workspace settings so this hopefully won't happen in future.